src/sagemaker/feature_store/feature_group.py

```diff
@@ -50,6 +52,7 @@
 from sagemaker.feature_store.feature_definition import (
     FeatureDefinition,
     FeatureTypeEnum,
+    ListCollectionType,
```

Comment:
To clarify, is it intended here to only have one "ListCollectionType" (list-like), so that the ingester can pass a collection of values, regardless the actual type in feature_definition is list/set/vector? Also let the service side to de-dupe if actual type is set?

Reply:
This is only used when the customers use load_feature_definitions method, to identify every collection type as a List. This is because we have no mechanism to distinguish between different collection types in a Panda DataFrame. Customer will have to manually update the feature definitions if they want to use other collection types. 

As for ingestion as you mentioned we only need to know if it's a collection type and we do that by making a describe feature group call before ingesting.